services/static-webserver/client/source/class/osparc/vipMarket/AnatomicalModelDetails.js

```diff
@@ -167,40 +167,42 @@ qx.Class.define("osparc.vipMarket.AnatomicalModelDetails", {
         row: 2,
       });
 
-      const buttonsLayout = new qx.ui.container.Composite(new qx.ui.layout.HBox(5));
-      if (anatomicalModelsData["leased"]) {
-        const leaseModelButton = new qx.ui.form.Button().set({
-          label: this.tr("3 seats Leased (27 days left)"),
-          appearance: "strong-button",
-          center: true,
-          enabled: false,
-        });
-        buttonsLayout.add(leaseModelButton, {
-          flex: 1
-        });
-      }
-      const leaseModelButton = new osparc.ui.form.FetchButton().set({
-        label: this.tr("Lease model (2 for months)"),
-        appearance: "strong-button",
-        center: true,
-      });
-      leaseModelButton.addListener("execute", () => {
-        leaseModelButton.setFetching(true);
-        setTimeout(() => {
-          leaseModelButton.setFetching(false);
-          this.fireDataEvent("modelLeased", this.getAnatomicalModelsData()["ID"]);
-        }, 2000);
-      });
-      buttonsLayout.add(leaseModelButton, {
-        flex: 1
-      });
-      cardLayout.add(buttonsLayout, {
-        column: 0,
-        row: 3,
-        colSpan: 2,
-      });
-
       return cardLayout;
     },
+
+    __createPricingUnits: function(anatomicalModelsData) {
+      console.log(anatomicalModelsData);
+      const pricingUnitsLayout = new qx.ui.container.Composite(new qx.ui.layout.HBox(5).set({
+        alignX: "center"
+      }));
+
+      const params = {
+        url: {
+          pricingPlanId: anatomicalModelsData["pricingPlanId"]
+        }
+      };
+      osparc.data.Resources.fetch("pricingPlans", "getOne", params)
+        .then(data => {
+          const pricingUnits = data["pricingUnits"];
+          pricingUnits.forEach(pricingUnit => {
+            const pUnit = new osparc.study.PricingUnit(pricingUnit).set({
+              showAwsSpecificInfo: false,
+              showUnitExtraInfo: false,
+              showEditButton: false,
+              showRentButton: true,
+              allowGrowY: false
+            });
+            pUnit.addListener("rentPricingUnit", () => this.__rentAnatomicalModel(anatomicalModelsData, pricingUnit));
+            pricingUnitsLayout.add(pUnit);
+          });
+        })
+        .catch(err => console.error(err));
+
+      return pricingUnitsLayout;
+    },
+
+    __rentAnatomicalModel: function(anatomicalModelsData, pricingUnit) {
+      console.log(":puschase", anatomicalModelsData["licensedItemId"], pricingUnit["pricingUnitId"]);
```

Comment:
"Puscas" was a real madrid player ;-)

Reply:
In this PR there is only one "real" and it's not madrid ;)